### DISCUSSION TITLE
Please make Form compatible with Storybook

### DISCUSSION THREAD

**Discussion by ianwremmel** [remix-run/remix#2512] [Proposals]
The `<Form/>` component throws an error if it's not nested under a `<RemixRoute />`. Unfortunately, `<RemixRoute/>` throws an error if it's not nested under a `<RemixEntry />`. `<RemixEntry/>` requires number of arguments that would be strange to mock for purposes of storybook. Would it be reasonable to export a component that sets up the underlying contexts so that `<Form />` can work in Storybook?

**Comment by petermekhaeil** [comment]
https://github.com/remix-run/remix/discussions/2481
